ext/standard/basic_functions.c

```diff
@@ -1182,6 +1182,7 @@ PHP_FUNCTION(time_nanosleep)
 		RETURN_TRUE;
 	} else if (errno == EINTR) {
 		array_init(return_value);
+		MSAN_UNPOISON(php_rem);
```

Comment:
is it related to a clang version ?

Reply:
Indeed. It seems to believe `nanosleep()` with `EINTR` does not initialize the structure, but it does.

https://man7.org/linux/man-pages/man2/nanosleep.2.html

> If the call is interrupted by a signal handler, nanosleep() returns -1, sets [errno](https://man7.org/linux/man-pages/man3/errno.3.html) to EINTR, and writes the remaining time into the structure pointed to by rem unless rem is NULL.